shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DDLStatement.g4

```diff
@@ -625,10 +625,13 @@ compoundStatement
     ;
 
 validStatement
-    : (createTable | alterTable | dropTable | truncateTable 
-    | insert | replace | update | delete | select | call
+    : (createTable | alterTable | dropTable | truncateTable | validDMLStatement
     | setVariable | beginStatement | declareStatement | flowControlStatement | cursorStatement | conditionHandlingStatement) SEMI_?
     ;
+    
+validDMLStatement
```

Comment:
> I suppose `validDMLStatement` is just for your convenience. If `MySQLVisitor` contains the content of `MySQLDMLStatement`, this rule is unneeded, right? As there is no such concept in the SQL grammar of MySQL, I guess.

@tristaZero Yes, the `validDMLStatement` is simply a way to differentiate the statements in routine Body so that they can be processed during routine parsing. If `MySQLVisitor` contains the content of `MySQLDMLStatement`, this rule can be removed.